### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Souvikns**
Changed the readme file and added a link under the testing section 

#### References to other Issues or PRs or Relevant literature
#235 (comment)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ For testing your patch locally follow the steps given below,
 For a good visualisation of the different data structures and algorithms, refer the following websites:
 - https://visualgo.net/
 - https://www.cs.usfca.edu/~galles/visualization/
+- [https://github.com/adavijit/BlogMan](https://github.com/adavijit/BlogMan)
```
Comment (czgdp1807) [inline README.md:41]: This is not the URL which was to be added. Please see the issue description of #264

Comment (Souvikns) [reply README.md]: Oh sorry, I thought I had the URL copied. 
I have made the changes.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ For testing your patch locally follow the steps given below,
 For a good visualisation of the different data structures and algorithms, refer the following websites:
 - https://visualgo.net/
 - https://www.cs.usfca.edu/~galles/visualization/
+- [https://opendatastructures.org/ods-python.pdf](https://opendatastructures.org/ods-python.pdf)
```
Comment (czgdp1807) [inline README.md:41]:
```suggestion

You can use the examples given in the following book as tests for your code:
- [https://opendatastructures.org/ods-python.pdf](https://opendatastructures.org/ods-python.pdf)
```

**Comment by czgdp1807** [comment]
@Souvikns Are you participating through GSSoC, 2020?

**Comment by Souvikns** [comment]
@czgdp1807  I knew about it recently, I am not participating in GSSoC, 2020

**Pull request merged**